index.html

```diff
@@ -38,5 +38,26 @@ <h2 class="text-container" id="sdsection">
       <img src="arrow-button.svg" class="arrow-button"><a href="#"></a></img>
       </div>
     </section>
+    <section class="works">
+      <h1 class="recent-works">My Recent Works</h1>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
```

Comment:
- [ ] Please make sure to style all your buttons (e.g., "Get my resume") to follow the button styling: Active, Hover, and Pressed :100: , You can refer to the Figma design to see how they are supposed to look in each state. :wink: